mgmtsystem/azure.py

```diff
@@ -297,6 +297,7 @@ def delete_stack(self, stack_name, resource_group=None):
             -DeploymentName \"{stack}\" -Force
             }}
             """.format(rg=resource_group or self.resource_group, stack=stack_name))
+        self.delete_vm(stack_name, resource_group or self.resource_group)
```

Comment:
Yes, I rewrote the stack data to read:

        stack_data = {
            'stack_name': stackname,
            'vm_name': stackname,
            .......